OGC-Core/OGCReferenceSystem.class.st

```diff
@@ -0,0 +1,20 @@
+Class {
+	#name : #OGCReferenceSystem,
+	#superclass : #Object,
+	#category : #'OGC-Core'
+}
+
+{ #category : #accessing }
+OGCReferenceSystem class >> axisName [
+	self subclassResponsibility 
+]
+
+{ #category : #accessing }
+OGCReferenceSystem class >> dimension [
+	self subclassResponsibility 
+]
+
+{ #category : #testing }
+OGCReferenceSystem class >> isAbstract [
+	^ true

```

Comment:
If you plan to have one abstract base class which has quite some concrete implementations you can do 

`^ self == OGCReferenceSystem`

This way it only counts for the abstract class and you don't have to overwrite that in all subclasses

Reply:
I would like here to forbid any instanciation of a Reference System class, and I consider all subclasses of `OGCReferenceSystem` as abstract

Reply:
But I'll adopt this practice for OGCGeometry and subclasses (where I only consider OGCGeometry, OGCSurface and OGCMultiCurve as abstract for now)